home/default.nix

```diff
@@ -3,33 +3,45 @@ with lib;
 
 let
 
-  cfg = config.colorschemes.kauz;
+  cfg = config.kauz;
+
+  isFish = cfg.fish.enable;
+  isKitty = cfg.kitty.enable;
+  isTmux = cfg.tmux.enable;
+  isNeovim = cfg.neovim.enable;
 
 in {
 
-  options.colorschemes.kauz = {
-    enable = mkEnableOption "a colorscheme for neovim, kitty, fish and tmux";
+  options.kauz = {
+    fish.enable =
+      mkEnableOption "Whether to enable the Kauz colorscheme for Fish";
+    kitty.enable =
+      mkEnableOption "Whether to enable the Kauz colorscheme for Kitty";
+    tmux.enable =
+      mkEnableOption "Whether to enable the Kauz colorscheme for Tmux";
+    neovim.enable =
+      mkEnableOption "Whether to enable the Kauz colorscheme for Neovim";
   };
 
-  config = mkIf cfg.enable {
-
-    programs.kitty.extraConfig = ''
-      include ${pkgs.kauz-kitty}/kauz.conf
-    '';
-
-    programs.tmux.extraConfig = builtins.readFile "${pkgs.kauz-tmux}/kauz.tmux";
-
+  config = mkIf isFish {
     programs.fish.plugins = [{
       name = "kauz-fish";
       inherit (pkgs.kauz-fish) src;
     }];
-
+  } // mkIf isKitty {
+    programs.kitty.extraConfig = ''
+      include ${pkgs.kauz-kitty}/kauz.conf
+    '';
+  } // mkIf isTmux {
+    programs.tmux.extraConfig = ''
+      builtins.readFile "${pkgs.kauz-tmux}/kauz.tmux";
```

Comment:
I don't think this will work inside a string.

Reply:
yes my bad